packages/ui/build/rollup/plugins/rollup-teaser-preset.js

```diff
@@ -1,4 +1,4 @@
-import { terser, Options } from 'rollup-plugin-terser'
```

Comment:
I got warning in console that Options  declared but not used, so I removed it
but now I see that it used to define type `{ Options }`
will revert

Reply:
It was used on 3d line as JSDoc.

Reply:
You can try to use `import type { Options }` here to prevent this warning. (Maybe)